<launch>
  <include file="$(find bumble)/config/odrive.launch"/>
  <include file="$(find bumble)/config/audio.launch"/>
  <include file="$(find bumble)/config/simplebgc.launch"/>
  <include file="$(find bumble)/config/camera.launch"/>
  <include file="$(find bumble)/config/apprewardbutton.launch"/>
  <include file="$(find bumble)/config/rosbag_capture.launch">
    <arg name="prefix" value="randomwalk"/>
  </include>

  <node name="randomwalk" pkg="bumble" type="randomwalk"
        required="true" output="screen">
       <param name="change_frequency_secs" value="2.0"/>

       <param name="pitch_angle_min" value="-35.0"/>
       <param name="pitch_angle_max" value="35.0"/>

       <param name="yaw_angle_min" value="-35.0"/>
       <param name="yaw_angle_max" value="35.0"/>

       <param name="forward_speed_min" value="-0.35"/>
       <param name="forward_speed_max" value="0.35"/>

       <param name="angular_speed_min" value="-0.50"/>
       <param name="angular_speed_max" value="0.50"/>
  </node>
</launch>